webapp/views/wpt-app.js

```diff
@@ -133,7 +133,8 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
                      path="{{subroute.path}}"
                      test-runs="{{testRuns}}"
                      test-paths="{{testPaths}}"
-                     search-results="{{searchResults}}"></wpt-results>
+                     search-results="{{searchResults}}"
+                     metadata="[[metadata]]"></wpt-results>
```

Comment:
metadata is fetched inside of wpt-metadata.js so it should be always empty at this level?